test/controllers/CheckYourAppealController.test.ts

```diff
@@ -83,28 +83,16 @@ describe('CheckYourAppealController', () => {
                 });
         });
 
-        it('should return 200 with no populated session data', async () => {
+        it('should return 500 when there is no appeal data', async () => {
```

Comment:
I guess that change in behaviour is fine as there was never requirement to display empty page and navigation system would not allow this to happen.  I even wonder whether we need test like that.

Reply:
the test was added when we first implemented auth and session middleware, i no longer feel it's relevant - i'll remove this